tools/data-handler/src/utils/clingo-parser.ts

```diff
@@ -130,7 +130,7 @@ class ClingoParser {
           break;
         case 'date':
         case 'dateTime':
-          res[fieldName] = new Date(parseInt(decoded)).toISOString();
+          res[fieldName] = new Date(parseInt(decoded) * 1000).toISOString();
```

Comment:
For safety it might be reasonable to provide the radix as a parameter (`parseInt(decoded, 10)`), this might avoid strange conversion issues in the future (somebody will eventually pass hex value here anyways). 
Is there any risk for overflow when multiplying the integer with 1000? What if it is already `maxInt` or close to it?

Reply:
No risk, because clingo can only handle 32-bit numbers.